_episodes/03-create.md

```diff
@@ -527,10 +527,10 @@ ls: cannot access 'quotes.txt': No such file or directory
 >
 > > ## Solution
 > > ```
-> > $ rm: remove regular file 'thesis_backup/quotations.txt'?
+> > $ rm: remove regular file 'thesis_backup/quotations.txt'? y
 > > ```
 > > {: .language-bash}
-> > The `-i` flag will prompt before every removal.
+> > The `-i` flag will prompt before (every) removal (use <kbd>Y</kbd> to confirm deletion or <kbd>N</kbd> to keep the file).
```

Comment:
We're a bit too explicit here, with capital Y and N, small y and n work also.

Reply:
I've shown the lower-case 'y' in the code block, but I think the keyboard representation is correct. Keyboards show the upper case letter, so I think this is clear as it can be that it's 'press y' rather than 'press shift + y'.

Reply:
Ah, I see. I have missed that.